packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts

```diff
@@ -130,25 +130,13 @@ export class BitcoinChainAdapter implements ChainAdapter<ChainTypes.Bitcoin> {
       }
 
       const path = toRootDerivationPath(bip32Params)
-      const publicKeys = await wallet.getPublicKeys([
-        {
-          coin: this.coinName,
-          addressNList: bip32ToAddressNList(path),
-          curve: 'secp256k1', // TODO(0xdef1cafe): from constant?
-          scriptType
-        }
-      ])
-
-      if (!(publicKeys ?? []).length) {
-        throw new Error('BitcoinChainAdapter: no public keys available from wallet')
-      }
-      const pubkey = publicKeys?.[0]?.xpub
+      const pubkey = await this.getPubKey(wallet, bip32Params)
       if (!pubkey) throw new Error('BitcoinChainAdapter: no pubkey available from wallet')
```

Comment:
We are already throwing this error essentially in `getPubKey` if we want to remove